pkg/storage/mvcc_key.go

```diff
@@ -287,3 +292,66 @@ func decodeMVCCTimestampSuffix(encodedTS []byte) (hlc.Timestamp, error) {
 	}
 	return decodeMVCCTimestamp(encodedTS[:encodedLen-1])
 }
+
+// MVCCRangeKey is an MVCC key span at a timestamp.
```

Comment:
nit: it sounds ambiguous mvcc implies a timestamp aready, maybe just a keyspan at a timestamp?